### ISSUE TITLE
Two minor issues in Dor Grestin

### ISSUE THREAD

**Issue by macce2** [Pyrdacor/Ambermoon-Advanced#20]
The dwarf lady tells you that Asrub, the healer lives 
on the north-west part of the town;
but the healer's house is actually located more on 
the north-east side of the town,
or better say north side.

The Nest (Inn):  despite the message that tells you
that the party is ready to leave after a good rest/sleep;
the party hasn't actually rested/slept at all,
so you have to select the option Sleep/rest
After the message appearing to actually get some sleep.
This should be corrected.

**Comment by Pyrdacor** [comment]
I think you mean it is actually located in the north east. This indeed should be fixed.

The issue with sleeping in inns is a general one. Resting in inns just means teleporting to the rest room and opening the inn screen. You have to sleep there and only then will recover.

But maybe this is not so bad. It is like you rent a room for the night. You can still decide to not sleep there and just leave in real life too.

**Comment by macce2** [comment]
> I think you mean it is actually located in the north east. This indeed should be fixed.
> 

Yes, I think one could say that Asrub's house is located on the north side of the
town.


> The issue with sleeping in inns is a general one. Resting in inns just means teleporting to the rest room and opening the inn screen. You have to sleep there and only then will recover.
> 
> But maybe this is not so bad. It is like you rent a room for the night. You can still decide to not sleep there and just leave in real life too.

I think the Nest could be the first one in the game so far, that left me a little
puzzled when entered the first time, because of the message
telling me that the party Has Had a good rest.
